libs/language-server/src/test/langium-utils.ts

```diff
@@ -0,0 +1,61 @@
+/** ****************************************************************************
+ * Copyright 2021 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
```

Comment:
To keep the structure of the REUSE header consistent:
```suggestion
// SPDX-FileCopyrightText: 2021 TypeFox GmbH
//
// SPDX-License-Identifier: MIT

```

Then, we need to add the MIT license to our `LICENSES` folder in the root of the repository. This can be done automatically by running `reuse download MIT` (requires the [REUSE tool](https://github.com/fsfe/reuse-tool) to be installed).